todo/tests/test_views.py

```diff
@@ -133,6 +133,64 @@ def test_no_javascript_in_task_note(todo_setup, client):
     assert task.note == bleach.clean(note, strip=True)
 
 
+@pytest.mark.django_db
+def test_completed_unchanged(todo_setup, client):
+
+    def makeTask(title, client):
```

Comment:
So `makeTask` is not a test, it's a fixture - a function used by pytest to set up mock data to feed to tests. But you didn't need to do that - we already had mock data being provided to tests, in `conftest.py`. So you just needed one actual test function, that called `todo_setup` as a fixture. Then you'd have access to the tasks already set up there. Or if you needed to add more to the fixtures, that would be the place to do that.

Reply:
Just wanted to say I did finally read this! Lol. Thank you for the time you invested in giving me feedback.